.circleci/config.yml

```diff
@@ -66,6 +66,11 @@ jobs:
       - restore_cache:
           keys:
           - v1-stack-dependencies-{{ checksum "postgrest.cabal" }}-{{ checksum "stack.yaml" }}
+      - run:
+          name: install ncat
```

Comment:
Is `nc` available in CircleCI? Maybe we could omit this dependency, try:

```bash
echo -e "GET / HTTP/1.0\n" | nc -U "/tmp/postgrest.sock"
```

Reply:
@steve-chavez I have tried this; for some reason this line works from the command line but I keep getting 400 bad requests when I execute it in a script. 

I have found a reference to [this askubuntu question](https://askubuntu.com/questions/493862/netcat-not-working-at-least-not-that-i-can-tell) but it does not seem to apply for io-tests.sh

any pointers?

Reply:
@Dansvidania  Oh. I think it's related to the shell. 

Are you executing the script like `./io-tests.sh`? If so, does it work if you try with `bash io-tests.sh`?

Reply:
@steve-chavez unfortunately not.

Reply:
@Dansvidania I see. I think for now we can leave the tests with nmap. Check my other comment about `sleep`.

Reply:
@Dansvidania Also, please  `rebase` against master. Just merged another PR that changed some of the files. Sorry about that.